vscode-extension/src/extension.ts

```diff
@@ -233,15 +227,27 @@ async function createNewAIConfig(
   const fileContentBuffer = await vscode.workspace.fs.readFile(fileContentPath);
   const initialContent = fileContentBuffer.toString();
 
-  const doc = await vscode.workspace.openTextDocument({
-    content: initialContent,
-    language: mode,
-  });
+  let doc: vscode.TextDocument = null;
+  let i = 0;
+  while (!doc) {
+    const fileName = `untitled${i === 0 ? "" : "-" + i}.aiconfig.${mode}`;
+    const filePath = workspaceUri
+      ? path.join(workspaceUri.path, fileName)
+      : fileName;
+    const untitledUri = vscode.Uri.file(filePath).with({ scheme: "untitled" });
+    try {
+      doc = await vscode.workspace.openTextDocument(untitledUri);
+      break;
+    } catch (err) {
+      i++;
+    }
+  }
+
+  const editor = await vscode.window.showTextDocument(doc, { preview: false });
```

Comment:
I like preview: false. I wish this was the default setting whenever opening docs so you don't have to double-click to keep file tabs sticky lol